src/DataModel/IData.ts

```diff
@@ -0,0 +1,6 @@
+import {IMetadata} from "./IMetadata";
+
+export interface IData extends Object
```

Comment:
Indeed I'd love to have raw data and hydra stuff some how separated. As for the _IData_ replacements - _IResource_ is already used to denote actual resource. I wouldn't like to add a metadata property there. Bu I see direction your thoughts are going - I need to give it some thoughs.
As for metadata - maybe IHypermedia. I've got feeling that it's still to narrow as there may be something that doesn't qualify.